lib/onc_certification_g10_test_kit/unrestricted_resource_type_access_group.rb

```diff
@@ -94,32 +94,11 @@ class UnrestrictedResourceTypeAccessGroup < Inferno::TestGroup
       oauth_credentials :smart_credentials
     end
 
-    ALL_RESOURCES =
-      [
-        'AllergyIntolerance',
-        'CarePlan',
-        'CareTeam',
-        'Condition',
-        'Device',
-        'DiagnosticReport',
-        'DocumentReference',
-        'Goal',
-        'Immunization',
-        'MedicationRequest',
-        'Observation',
-        'Procedure',
-        'Patient',
-        'Provenance',
-        'Encounter',
-        'Practitioner',
-        'Organization'
-      ].freeze
-
-    V5_ALL_RESOURCES = (ALL_RESOURCES + ['ServiceRequest']).freeze
+    V5_EXCLUDED_RESOURCES = ['RelatedPerson'].freeze
 
-    V6_ALL_RESOURCES = (V5_ALL_RESOURCES + ['Coverage', 'MedicationDispense']).freeze
+    V6_EXCLUDED_RESOURCES = (V5_EXCLUDED_RESOURCES + ['Specimen']).freeze
 
-    V7_ALL_RESOURCES = (V6_ALL_RESOURCES + ['Location']).freeze
+    V7_EXCLUDED_RESOURCES = V6_EXCLUDED_RESOURCES.freeze
```

Comment:
You already froze it in line 99.

```suggestion
    V7_EXCLUDED_RESOURCES = V6_EXCLUDED_RESOURCES
```